gui/choc_WebView.h

```diff
@@ -520,6 +523,17 @@ struct choc::ui::WebView::Pimpl
     bool loadedOK() const           { return getViewHandle() != nullptr; }
     void* getViewHandle() const     { return (CHOC_OBJC_CAST_BRIDGED void*) webview; }
 
+    bool clearCookies()
+    {
+        auto store = objc::call<id> (objc::getClass ("WKWebsiteDataStore"), "defaultDataStore");
+        auto allTypes = objc::call<id> (objc::getClass ("WKWebsiteDataStore"), "allWebsiteDataTypes");
+        objc::call<void>( store, "fetchDataRecordsOfTypes:completionHandler:", allTypes, ^(id records)
+            {
+                objc::call<void> (store, "removeDataOfTypes:forDataRecords:completionHandler:", allTypes, records, ^(void){});
```

Comment:
Is there any reason we might need to wait for that completion handler callback?

Reply:
I guess technically the cookies won't be cleared until that callback is called, though we don't get any confirmation from it that anything was successful. do you think it's worth adding a callback to `clearCookies`?

Reply:
Hmm I suppose the only real issue would be a race condition if the caller thinks cookies are cleared but they haven't actually been cleared yet. It seems like super low likelihood... I think it's probably fine